config/config.toml.example

```diff
@@ -0,0 +1,167 @@
+# A sample configuration for fedora-messaging. This file is in the TOML format.
+# For complete details on all configuration options, see the documentation
+# https://fedora-messaging.readthedocs.io/en/latest/configuration.html.
+
+amqp_url = "amqp://"
+
+publish_exchange = "amq.topic"
+
+callback = "hotness.consumers:BugzillaTicketFiler"
+
+# Note the double brackets below.
+# To add another binding, add another [[bindings]] section.
+[[bindings]]
+queue = "anitya"
+exchange = "amq.topic"
+routing_keys = [
+    "org.release-monitoring.prod.anitya.project.version.update",
+    "org.release-monitoring.prod.anitya.project.map.new",
+]
+
+[[bindings]]
+queue = "koji"
+exchange = "amq.topic"
+routing_keys = [
+    "org.fedoraproject.prod.buildsys.task.state.change",
+]
+
+[tls]
+ca_cert = "/etc/pki/tls/certs/ca-bundle.crt"
+keyfile = "/my/client/key.pem"
+certfile = "/my/client/cert.pem"
+
+[client_properties]
+app = "the-new-hotness"
+
+[exchanges.custom_exchange]
+type = "fanout"
+durable = false
+auto_delete = false
+arguments = {}
+
+[queues.anitya]
+durable = true
+auto_delete = false
+exclusive = false
+arguments = {}
+
+[queues.koji]
+durable = true
+auto_delete = false
+exclusive = false
+arguments = {}
```

Comment:
Any reason to define two separate queues here? I think you could define only one, call it "the-new-hotness" for example (I don't remember if dashes are allowed), and bind all the topics to it.

Queues are usually named after the consumer that read them, not after the producer.

Reply:
I named them after producer, but this is probably not good approach. I will change this.